my-app-v/src/api/api.js

```
@@ -0,0 +1,63 @@
+import axios from "axios";
+const API_URL = "http://localhost:3001/todos";
+
+export const fetchTodos = async () => {
+    try{
+        const response = await axios.get(API_URL);
+        return response.data;
+    }
+    catch (error) {
+        handlesError(error);
+        return [];
```

[cc] why to return empty []

---

If fetchTodos doesn't return data or fails, it should ensure that todos is always an empty array ([]) instead of undefined, to prevent errors when using .map().

---

it should return the error only